crates/ruff/src/rules/ruff/rules/ambiguous_unicode_character.rs

```diff
@@ -1698,45 +1698,47 @@ pub fn ambiguous_unicode_character(
     let mut col_offset = 0;
     let mut row_offset = 0;
     for current_char in text.chars() {
-        // Search for confusing characters.
-        if let Some(representant) = CONFUSABLES.get(&(current_char as u32)) {
-            if !settings.allowed_confusables.contains(&current_char) {
-                let col = if row_offset == 0 {
-                    start.column() + col_offset
-                } else {
-                    col_offset
-                };
-                let location = Location::new(start.row() + row_offset, col);
-                let end_location = Location::new(location.row(), location.column() + 1);
-                let mut diagnostic = Diagnostic::new::<DiagnosticKind>(
-                    match context {
-                        Context::String => AmbiguousUnicodeCharacterString {
-                            confusable: current_char,
-                            representant: *representant as char,
+        if !current_char.is_ascii() {
+            // Search for confusing characters.
+            if let Some(representant) = CONFUSABLES.get(&(current_char as u32)).copied() {
```

Comment:
Looks like also changing to `.copied` here 👍

Reply:
ah yeah, I should have mentioned it in the summary.